From 0c048d1101db968246cedaeeb0212360e54259b0 Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke@hauke-m.de>
Date: Sat, 27 Jun 2026 17:29:48 +0200
Subject: [PATCH 135/138] umdns: update to Git HEAD (2026-06-27)

dbee55098c2e Change domain variable to const char* for gcc 15 compatibility

(cherry picked from commit f5d928e52a5f8c3d236dc0602fb407a2d3ec0285)
Link: https://github.com/openwrt/openwrt/pull/25589
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>
---
 package/network/services/umdns/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/package/network/services/umdns/Makefile b/package/network/services/umdns/Makefile
index 40ee5ed492..d317fac304 100644
--- a/package/network/services/umdns/Makefile
+++ b/package/network/services/umdns/Makefile
@@ -12,9 +12,9 @@ PKG_RELEASE:=1
 
 PKG_SOURCE_URL=$(PROJECT_GIT)/project/mdnsd.git
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_DATE:=2026-06-16
-PKG_SOURCE_VERSION:=1b5e7bf1cec775b89f1a6068dc0b9df3593b5986
-PKG_MIRROR_HASH:=7d9ec860af2b8217f65cd4af92a61be1925ad171dee9b536c4914cc0080e3303
+PKG_SOURCE_DATE:=2026-06-27
+PKG_SOURCE_VERSION:=dbee55098c2e7a0a76feec414a11d76838572918
+PKG_MIRROR_HASH:=726e7874f7399fd286d81479ff00cb1cdd231cf3a25b8f89e4a9a1c45bb670bc
 
 PKG_MAINTAINER:=John Crispin <john@phrozen.org>
 PKG_LICENSE:=LGPL-2.1
-- 
2.34.1

